### PR TITLE
Update tasks wraper to include api call for logs for a run

### PR DESCRIPTION
**Pull request by Palakp41**

### CODE REVIEW

File: src/wrappers/tasks.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Run, Task, TasksApi, User } from "../api";
+import { LogEvent, Logs, Run, Task, TasksApi, User } from "../api";
```
Comment (bthesorceror) [inline src/wrappers/tasks.ts]: Logs is imported but never used

File: src/wrappers/tasks.ts [hunk 2]
```diff
@@ -107,6 +121,17 @@ export default class {
     return data;
   }
 
+  public async getLogsByRunID(
```
Comment (bthesorceror) [inline src/wrappers/tasks.ts]: Should this be `getLogEventsByRunID`?

File: src/wrappers/tasks.ts (hunk 1 above)
Comment (Palakp41) [reply src/wrappers/tasks.ts]: I removed it from the import now

File: src/wrappers/tasks.ts (hunk 2 above)
Comment (Palakp41) [reply src/wrappers/tasks.ts]: Yes, I updated the name

**Pull request merged**